packages/amplify-e2e-tests/src/__tests__/schema-auth-11-a.test.ts

```diff
@@ -10,7 +10,7 @@ describe('api directives @auth batch 3a', () => {
   let projectDir: string;
 
   beforeEach(async () => {
-    projectDir = await createNewProjectDir('auth3');
+    projectDir = await createNewProjectDir('auth11a');
```

Comment:
The directory names should align with the test file name, so I updated these

Reply:
we do generate random uuids after the project name - so these are just suffixes

Reply:
Ok, instead of manually updating these, we could create a helper function that does this for us. Something like
```typescript
function getBasename(filename = ''): string {
  const filename = path.basename(filename);
  const extensionIndex = filename.lastIndexOf('.')
  if (extensionIndex < 0) {
      return filename ?? 'test';
  }
  return filename.slice(0, extensionIndex);
}
projectDir = await createNewProjectDir(getBasename(__filename));
```

Reply:
I really like this idea, but we'll need to find a way to isolate that into a PR, it will affect a lot of files (300ish)